students/lakatorz_izaak/lesson_09_reading_and_writing_files/regex_search.py

```diff
@@ -0,0 +1,18 @@
+import os
+import re
+
+
+user_regex = re.compile(r'\d')
+
+path = '../lesson_09_reading_and_writing_files/'
+
+folder_files = os.listdir(path)
+
+txt_files = [x for x in folder_files if x.endswith(".txt")]
+
+for filename in txt_files:
+    file = open(path + filename)
```

Comment:
Możesz w takich sytuacjach użyć `os.path`: https://docs.python.org/3/library/os.path.html
Ta libka ułatwia manipulacje ścieżkami, m.in. częsty problem z brakiem lub obecnością slasha w ścieżce do pliku:
```
> os.path.join('path/to/','file')
=> 'path/to/file'
> os.path.join('path/to','file')
=> 'path/to/file'
```